src/models/databases_connectors/databases_access.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+interface DatabasesAccess
+{
+	public function isExcerciceExist(int $id): bool;
```

Comment:
doesExerciceExist would make more sense

Reply:
"Does" is used to describe actions and is the third person singular form of the verb "do". "Is" describes the condition or existence of a subject and is the third person singular form of the verb "be".